src/components/download-print-menu/download-print-menu.tsx

```diff
@@ -0,0 +1,171 @@
+import { h, Component } from "preact";
+import { Popover } from "@playkit-js-contrib/ui";
+import { PopoverMenu, PopoverMenuItem } from "../popover-menu";
+import * as styles from "./download-print-menu.scss";
+
+interface DownloadPrintMenuProps {
+    dropdownAriaLabel: string;
+    printButtonAriaLabel: string;
+    downloadButtonAriaLabel: string;
+    onDownload: () => void;
+    onPrint: () => void;
+    downloadDisabled: boolean;
+    printDisabled: boolean;
+}
+
+interface DownloadPrintMenuState {
+    popoverOpen: boolean;
+}
+
+export class DownloadPrintMenu extends Component<DownloadPrintMenuProps, DownloadPrintMenuState> {
+    static defaultProps = {
+        dropdownAriaLabel: "Download or print transcript",
+        printButtonAriaLabel: "Print current transcript",
+        downloadButtonAriaLabel: "Download current transcript"
+    };
+    state: DownloadPrintMenuState = {
+        popoverOpen: false
+    };
+    private _controlElement: any;
+
+    componentDidMount() {
+        document.addEventListener("click", this._handleClickOutside);
+    }
+
+    componentWillUnmount() {
+        document.removeEventListener("click", this._handleClickOutside);
+    }
+
+    // TODO: could be used for popover accessibility
+    // componentDidUpdate(prevProps: DownloadPrintMenuProps, prevState: DownloadPrintMenuState) {
+    //     if (!prevState.popoverOpen && this.state.popoverOpen) {
+    //         this._controlElement.children[0].focus();
+    //     }
+    // }
+
+    private _handleClickOutside = (e: any) => {
+        if (
+            !!this._controlElement &&
+            !this._controlElement.contains(e.target) &&
+            this.state.popoverOpen
+        ) {
+            this._handleClosePopover();
+        }
+    };
+
+    private _handleClosePopover = () => {
+        this.setState({ popoverOpen: false });
+    };
+
+    private _onButtonClick = () => {
+        this.setState((state: DownloadPrintMenuState) => ({ popoverOpen: !state.popoverOpen }));
+    };
+
+    private _onDownloadClicked = () => {
+        this._handleClosePopover();
+        this.props.onDownload();
+    };
+
+    private _onPrintClicked = () => {
+        this._handleClosePopover();
+        this.props.onPrint();
+    };
+    private _onKeyDown = (e: KeyboardEvent, cb: Function) => {
```

Comment:
@semarche-kaltura took me a minute to decode CB as callback... 
We can use longer readable names

Reply:
done